beta/src/pages/learn/add-react-to-a-website.md

```diff
@@ -176,99 +176,99 @@ return React.createElement(
 
-The watcher will create a preprocessed **like_button.js** with the plain JavaScript code suitable for the browser.
+वाचर ब्राउज़र के लिए उपयुक्त सादे जावास्क्रिप्ट कोड के साथ एक प्रीप्रोसेस्ड **like_button.js** बनाएगा।
 
 <Gotcha>
 
-If you see an error message saying "You have mistakenly installed the `babel` package", you might have missed [the previous step](#add-jsx-to-a-project). Perform it in the same folder, and then try again.
+यदि आपको यह कहते हुए एक त्रुटि संदेश दिखाई देता है कि "आपने गलती से `babel` पैकेज सेटअप कर लिया है", तो आप [पिछला स्टेप](#add-jsx-to-a-project) चूक गए होंगे। इसे उसी फ़ोल्डर में निष्पादित करें, और फिर पुन: प्रयास करें।
 
 </Gotcha>
 
-As a bonus, this also lets you use modern JavaScript syntax features like classes without worrying about breaking older browsers. The tool we just used is called Babel, and you can learn more about it from [its documentation](https://babeljs.io/docs/en/babel-cli/).
+एक बोनस के रूप में, यह आपको पुराने ब्राउज़रों को तोड़ने की चिंता किए बिना classes जैसी आधुनिक जावास्क्रिप्ट सिंटैक्स सुविधाओं का उपयोग करने देता है। हमारे द्वारा अभी-अभी उपयोग किए गए टूल को babel कहा जाता है, और आप इसके बारे में [इसके दस्तावेज़ीकरण](https://babeljs.io/docs/en/babel-cli/) से अधिक जान सकते हैं।
 
-If you're getting comfortable with build tools and want them to do more for you, [we cover some of the most popular and approachable toolchains here](/learn/start-a-new-react-project).
+यदि आप बिल्ड टूल्स के साथ सहज हो रहे हैं और चाहते हैं कि वे आपके लिए और अधिक करें, [हम यहां कुछ सबसे लोकप्रिय और पहुंचने योग्य टूलचेन को कवर करते हैं](/learn/start-a-new-react-project)।
 
-<DeepDive title="React without JSX">
+<DeepDive title="React बिना JSX के">
 
-Originally JSX was introduced to make writing components with React feel as familiar as writing HTML. Since then, the syntax has become widespread. However, there may be instances where you do not want to use or cannot use JSX. You have two options:
+कोर रूप से JSX को React के साथ लेखन कौम्पोनॅन्ट को HTML लिखने के रूप में परिचित बनाने के लिए पेश किया गया था। तब से, वाक्यविन्यास व्यापक हो गया है। हालाँकि, ऐसे उदाहरण हो सकते हैं जहाँ आप JSX का उपयोग नहीं करना चाहते हैं या नहीं कर सकते हैं। आपके पास दो विकल्प हैं:
 
-- Use a JSX alternative like [htm](https://github.com/developit/htm) which doesn't use a compiler—it uses JavaScript's native Tagged Templates.
-- Use [`React.createElement()`](/reference/createelement), which has a special structure explained below.
+- [htm](https://github.com/developit/htm) जैसे JSX विकल्प का उपयोग करें, जो कंपाइलर का उपयोग नहीं करता है—यह जावास्क्रिप्ट के कोर टैग किए गए टेम्प्लेट का उपयोग करता है।
+- [`React.createElement()`](/reference/createelement) का उपयोग करें, जिसकी एक विशेष संरचना नीचे बताई गई है।
 
-With JSX, you would write a component like so:
+JSX के साथ, आप एक कौम्पोनॅन्ट लिखेंगे जैसे::
 
 ```jsx
 function Hello(props) {
-  return <div>Hello {props.toWhat}</div>;
+  return <div>नमस्ते {props.toWhat}</div>;
```

Comment:
`return <div>नमस्ते {props.toWhat}</div>;` => `return <div>Hello {props.toWhat}</div>;`

lets not change codes